torch/nn/parallel/distributed.py

```diff
@@ -199,6 +199,11 @@ class DistributedDataParallel(Module):
                                        part of this graph are preemptively marked
                                        as being ready to be reduced.
                                        (default: ``True``)
+        accumulate_grads (bool): when setting to ``True``, it allows model to
```

Comment:
Would I be correct if I assume that `optimizer.step()` should not be called when `accumulate_grads=True`?